hs_core/tests/api/native/test_publish_resource.py

```diff
@@ -395,7 +395,7 @@ def _get_expected_crossref_xml(res_id, timestamp, created_date, updated_date, pu
         <description>This is a test abstract</description>
         <fr:program name="fundref">
           <fr:assertion name="fundgroup">
-            <fr:assertion name="funder_name">National Science Foundation<fr:assertion name="funder_identifier">https://doi.org/10.13039/100000001</fr:assertion></fr:assertion>
+            <fr:assertion name="funder_name">National Science Foundation<fr:assertion name="funder_identifier">https://nsf.gov</fr:assertion></fr:assertion>
```

Comment:
```suggestion
            <fr:assertion name="funder_name">National Science Foundation<fr:assertion name="funder_identifier">https://ror.org/021nxhr62</fr:assertion></fr:assertion>
```
You will need to update the `get_funder_id` function so that ☝️ this suggestion is true during tests

Reply:
upon re-review, this suggestion still needs to be addressed.
`get_funder_id` still needs to be updated so that, in this case, it returns 
https://ror.org/021nxhr62 
not 
https://nsf.gov